### ISSUE TITLE
Fine-tune on my dataset（caffe）

### ISSUE THREAD

**Issue by Franciszzj** [terrychenism/ResNeXt#7]
when I fine-tune on my dataset with ResNeXt，I found it didn‘t work well。I GUESS it is BN layer result this.
Can you share your BN formula（expression）to me？
THX！

**Comment by Franciszzj** [comment]
@terrychenism The bn layer is same with Batch **Re**normalization？

**Comment by terrychenism** [comment]
No, bn layer is same with Batch Normalization

**Comment by Franciszzj** [comment]
And when I visualize the feature map at different layer，it‘s all dark! There is only little feature that have learned.

**Comment by terrychenism** [comment]
I didn't visualize the feature map.  did you try this model on imagenet?

**Comment by Franciszzj** [comment]
yes. 
when I delete bn layer, then use this model to visualize, it looks better.
My device is TITAN X（CUDA 8.0, cudnn v5.1）

**Comment by terrychenism** [comment]
Great!  plz make sure  use_global_stats: true

**Comment by Franciszzj** [comment]
at TRAIN and TEST phase, I both use true.
when i fine-turn this network, I make the first two stage's mult_lr = 0,other three stages 1. just like ResNet-101.
 I send my pt file to your email.

**Comment by Franciszzj** [comment]
I could not find your email @terrychenism Could you show me? My email is franciszzj@gmail.com

**Comment by terrychenism** [comment]
terrychen2012@live.com

**Comment by Franciszzj** [comment]
@terrychenism Thank you for your reply, I will try some other methods.
